backend/alert/templates/alert/email_close_alert.html

```diff
@@ -6,7 +6,7 @@
         <img src="https://penncoursealert.com/assets/pca/PCA_logo.png" alt="Penn Course Alert Logo" style="width: 40px" />
```

Comment:
Sort of unrelated, but we may want to factor out the email template and use template inheritance so that we don't have to copy/paste between the two. Just a bit more DRY.